chart/dapr-dashboard/README.md

```diff
@@ -0,0 +1,33 @@
+## Dapr Dashboard options
```

Comment:
please add the helm url to add to get the chart.

Reply:
Done.